### PR TITLE
Fix failing SCSS compilation

### PR DESCRIPTION
**Pull request by raimund-schluessler**
This should fix the failing SCSS compilation by server due to missing color variables.

I built the app, so testing is easier.
[tasks.tar.gz](https://github.com/nextcloud/tasks/files/5359927/tasks.tar.gz)

Please check this PR @LouisVallat @paulcalabro

### CODE REVIEW

**Comment by LouisVallat** [comment]
Thanks for the mention, I'm glad I could help :grinning: 
I just started a vanilla ``nextcloud`` image with Docker, set up my account and installed Tasks, checked that under NC20 it was still broken (that's normal, but just to set a baseline). I then removed everything under the ``custom_apps/tasks`` folder and extracted the files from the archive you conveniently provided (thank you by the way) in that folder, restarted the container, then opened another app such as ``Files`` in order to check that the server was all good and finally tried opening the Tasks app, successfully! The CSS is all there and the page isn't malformed.
I also checked the logs and didn't find anything related to this issue.
It's all good for me then! :100:

**Comment by raimund-schluessler** [comment]
@LouisVallat Great! I will wait for a bit more feedback and release a fixed version to the appstore tomorrow if everything goes well.

**Comment by Clewsy** [comment]
Hi @raimund-schluessler - confirming your PR worked for me too - tasks no longer malformed.  Thanks!

**Comment by lyallp** [comment]
> 
> 
> This should fix the failing SCSS compilation by server due to missing color variables.
> 
> I built the app, so testing is easier.
> [tasks.tar.gz](https://github.com/nextcloud/tasks/files/5359927/tasks.tar.gz)
> 
> Please check this PR @LouisVallat @paulcalabro

The supplied archive worked on my Nextcloud 20 installation. I noticed the max version of nextcloud had upped to 21 but the version of tasks had remained unchanged. I also noticed quite a few other changes that appeared to be to do with calendar deletes.
To that end, my testing involved disable/delete of my hacked copy, install/activate of the tar, test, disable/delete and restoration/activation of my hacked version until a new official release occurs.
Bottom line, the archive worked for me.

**Comment by paulcalabro** [comment]
Thanks! Will check later tonight once I get home.

**Comment by raimund-schluessler** [comment]
The appstore is down at the moment, hence the failing tests. Will merge anyway.
